public/swagger.v1.json

```diff
@@ -1819,6 +1819,12 @@
             "in": "path",
             "required": true
           },
+          {
```

Comment:
This should not be edited manually but generated base on code comments at start of ListIssues func.
Once the comments are added. Use `generate-swagger` to re-generate this file.

Reply:
Ah ok, I didn't know that. I'll do ASAP

Reply:
Done.